fastapi_pagination/ext/async_sqlalchemy.py

```diff
@@ -18,7 +18,7 @@ async def paginate(
     *,
     additional_data: AdditionalData = None,
     unique: bool = True,
-) -> AbstractPage[Any]:
+) -> TAbstractPage[Any]:
```

Comment:
Unfortunately, it won't work cause the python typing system doesn't support generic type-vars.

As an option, we can change all the pagination models to return `typing.Any` it will allow writing code like this:
```py
page: Page[User] = paginate(conn, query)
```

Reply:
what do you mean by > typing system doesn't support generic type-vars ?

Reply:
https://github.com/python/typing/issues/548

Reply:
@10ourto I will fix it by setting `Any` as the return type annotation for all `pagination` functions.

Reply:
I've used this snippet
```python
from fastapi_pagination.ext.async_sqlalchemy import paginate as lib_paginate

async def paginate(
    conn: AsyncSession,
    query: Select,
    params: Optional[AbstractParams] = None,
    *,
    additional_data: AdditionalData = None,
    unique: bool = True
) -> TAbstractPage:
    # we need this to overwrite original paginate return type
    return await lib_paginate(conn, query, params, additional_data=additional_data, unique=unique)
```
to fix it in my project

`TAbstractPage[Any]` I guess we do not need `[Any]` here since we expect here just any subclass of `AbstractPage`

Reply:
This code is incorrect from a `mypy` perspective, it will fix `PyCharm` errors because it will cast it to `Any`.
`mypy` error - https://github.com/uriyyo/fastapi-pagination/actions/runs/4024063860/jobs/6915698921#step:7:32

`TAbstractPage` is unbound `TypeVar` and it's not a valid case.

Reply:
but seems `TAbstractPage = TypeVar("TAbstractPage", bound="AbstractPage[Any]")` it is bound